modules/cudn-status-conditions.adoc

```diff
@@ -80,3 +80,157 @@ h|Message
+|You must set the `mtu` higher than `576`
+
+|The `mtu` must be at least `1280` when using the IPv6 subnet.
+|`MTU should be greater than or equal to 1280 when an IPv6 subnet is used`
+|You must set the `mtu` higher than or equal to `1280` when you have an IPv6 subnet defined on your user-defined network configuration.
+|===
+
+.Invalid `PhysicalNetworkName` scenarios types for the `ClusterUserDefinedNetwork` CR
+[cols="2a,2a,3a,3a",options="header"]
+|===
+
+|Condition type
+3+|Reason, Message, Resolution
+
+.6+|`invalid PhysicalNetworkName`
+3+|One of the following messages is returned when the `PhysicalNetworkName` is set incorrect:
+h|Reason
+h|Message
+h|Resolution
+
+|The name of the physical network is not set.
+|`spec.network.localnet.physicalNetworkName: Required value`
+|You must set the `physicalNetworkName` field.
+
+|The name of the physical network does not meet minimum length requirements.
+|`spec.network.localnet.physicalNetworkName in body should be at least 1 chars long`
+|You must set physical network name to be at least one character in length.
+
+|The name of the physical network exceeds the maximum character limit of 253.
+|`spec.network.localnet.physicalNetworkName: Too long: may not be more than 253 bytes`
+|You must set physical network name to not exceed the 253 character in length.
+
+|The name of the physical network must not contain `,` or `:`.
+|`physicalNetworkName cannot contain "," or ":" characters`.
+|You must remove the `,` or `:` from the physical network name.
+|===
+
+.Invalid `role` scenarios types for the `ClusterUserDefinedNetwork` CR
+[cols="2a,2a,3a,3a",options="header"]
+|===
+
+|Condition type
+3+|Reason, Message, Resolution
+
+.6+|`role unset` or `role is primary`
+3+|One of the following messages is returned when the `spec.network.localnet.role` is set incorrect:
+h|Reason
+h|Message
+h|Resolution
+
+|The `role` field must be set for your localnet topology.
+|`spec.network.localnet.role: Required value`
+|You must set the `role` field.
+
+|`Primary` is not a supported value for the Localnet topology.
+|`spec.network.localnet.role: Unsupported value: "Primary": supported values: "Secondary"`
+|You must set the `role` for your Localnet topology to `Secondary`-the accepted value.
+|===
+
+.Invalid `subnets` and `ipam` scenarios types for the `ClusterUserDefinedNetwork` CR
+[cols="2a,2a,3a,3a",options="header"]
+|===
+
+|Condition type
+3+|Reason, Message, Resolution
+
+.8+|`LocalnetInvalidSubnets`
+3+|One of the following messages is returned when either the `spec.network.localnet.subnets` or `spec.network.localnet.ipam` is set incorrect:
+h|Reason
+h|Message
+h|Resolution
+
+|The optional fields, `subnets` and `ipam.mode`, have to be set together.
+|`Subnets is required with ipam.mode is Enabled or unset, and forbidden otherwise`
+|You must set the `subnets` field when the `ipam.mode` is `Enabled` or when the IPAM mode is unset as it defaults to `Enabled`.
```

Comment:
I like how this reads, but I kind of think that the sentence in line 193 is simpler - i.e. "You must set the `spec.network.localnet.subnets` field unless the `spec.network.localnet.ipam.mode` is explicitly disabled"

FWIW, both are OK, I would buy either.